manim/cli/render/commands.py

```diff
@@ -136,4 +137,17 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version or args["notify_outdated_version"]:
+        manim = requests.get("https://pypi.org/pypi/manim/json")
+        releases = manim.json()["releases"].keys()
```

Comment:
Two comments regarding these two lines:

- I'm not particularly fond of a variable named `manim` and would prefer if it were renamend (I won't block the PR over it though),
- Given that `requests.get` raises an exception when the request fails (`pypi.org` could be down, or no internet connection could be available), I think it should be handled and a warning should be logged instead informing the user that it could not be checked whether the installed version is the latest. (I think this is actually important and I'd like to have that implemented before merging.)